### PR TITLE
first attempt pymongo lamda to extract civ stats

### PR DESCRIPTION
**Pull request by jdduprey**

### CODE REVIEW

File: extract-stats/civs_stats.py [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+def civ_stats_pipeline(start_date, end_date):
+    pipeline =[
+    {
+        '$match': {
+            'matchDate': {
+                '$gte': start_date, 
+                '$lt': end_date
+            }
+        }
+    }, {
```
Comment (erin-fitzpatric) [inline extract-stats/civs_stats.py:27]: These two $match opperations should be able to be combined into one.

File: extract-stats/civs_stats.py [hunk 2]
```diff
@@ -0,0 +1,535 @@
...
+                ]
+            }, 
+            'matchHistoryMap.player1': {
+                '$arrayElemAt': [
+                    '$matchHistoryArray', 1
+                ]
+            }
+        }
+    }, {
+        '$addFields': {
```
Comment (erin-fitzpatric) [inline extract-stats/civs_stats.py:74]: Can these multiple $addFields be combined?

**Review by erin-fitzpatric — CHANGES_REQUESTED** [review]

File: extract-stats/civs_stats.py (hunk 1 above)
Comment (jdduprey) [reply extract-stats/civs_stats.py:27]: done

File: extract-stats/civs_stats.py (hunk 2 above)
Comment (jdduprey) [reply extract-stats/civs_stats.py:74]: Not as is. These stages are a bit hacky but the only way I could figure out in MQL syntax to get the opponents civ into a playes match record. I couldn't figure out how to assign array[0]["civ_id"] to array[1]["civ_id"] so I converted them to individual fields...

**Review by erin-fitzpatric — APPROVED** [review]

**Pull request merged**